### ISSUE TITLE
Long README, move to wiki 

### ISSUE THREAD

**Issue by ethanmusser** [KodlabPenn/kodlab_mjbots_sdk#46]
The `README.md` is getting kind of long and difficult to navigate.  It might be time to break this out into several wiki pages.  Something like:

1. Setup
    1. Raspberry Pi 
    2. Laptop 
        1. Toolchain
        2. LCM
    3. Motors
2. Usage
    1. `RobotBase`
        1. Accessing Robot State
        2. User-Defined Robots
    2. `MjbotsControlLoop`
        1. LCM Logging
        2. LCM Input
        3. Control Loop Options (summary of options, including soft start)
    3. `Behavior` & `MjbotsBehaviorLoop` (when ready)
3. Building & Deploying Code

**Comment by jdcaporale** [comment]
I agree and was going to bring this up. This structure looks good to me
